homeassistant/components/unifi_access/lock.py

```diff
@@ -0,0 +1,75 @@
+"""Lock entities for the UniFi Access integration."""
+
+from typing import Any
+
+import uiaccessclient
+
+from homeassistant.components.lock import LockEntity
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import UniFiAccessConfigEntry, UniFiAccessDoorCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: UniFiAccessConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Configure lock entities."""
+    api_client = entry.runtime_data.api_client
+    door_coordinator = entry.runtime_data.door_coordinator
+
+    async_add_entities(
+        UniFiAccessDoorLock(hass, api_client, door_coordinator, door_id)
+        for door_id in door_coordinator.data
+    )
+
+
+class UniFiAccessDoorLock(CoordinatorEntity, LockEntity):
```

Comment:
```suggestion
class UniFiAccessDoorLock(CoordinatorEntity[..], LockEntity):
```
Please add the coordinator type here

Reply:
This is fixed.